third_party/rust/chromium_crates_io/Cargo.toml

```diff
@@ -26,6 +26,7 @@ adblock-cxx = { version = "1" }
 constellation-cxx = "0.1"
 challenge-bypass-ristretto-cxx = "1"
 zcash = "1"
+subtle = "2.6.1"
```

Comment:
Why do we need direct dependency on subtle here?

Reply:
We don't. Removed.